TFSIntegration/Script.ps1

```diff
@@ -0,0 +1,1609 @@
+            set { time = value; }
+        }
+
+        [XmlElement(ElementName = "testcase")]
+        public List<RunItem> RunItems { get; set; }
+
+        public void IncErrors()
+        {
+            this.Errors++;
+            IncTotal();
+        }
+        public void IncPassed()
+        {
+            this.Passed++;
+            IncTotal();
+        }
+        public void IncFailed()
+        {
+            this.Failed++;
+            IncTotal();
+        }
+
+        private void IncTotal()
+        {
+            this.Total++;
+        }
+
+    }
+
+    public class RunItem
+    {
+        public RunItem() { }
+
+        public RunItem(string caseTitle, string flowStatus, double elapsed, string schedule)
+        {
+            FlowTitle = caseTitle;
+            FlowStatus = flowStatus;
+            ElapsedTime = elapsed;
+            classname = schedule;
+            failure = null;
+        }
+
+        public RunItem(string caseTitle, string flowStatus, double elapsed, string stacktrace, string schedule)
+        {
+            FlowTitle = caseTitle;
+            FlowStatus = flowStatus;
+            ElapsedTime = elapsed;
+            failure = new Failure(stacktrace);
+            classname = schedule;
+        }
+        [XmlAttribute(AttributeName = "name")]
+        public string FlowTitle { get; set; }
+
+        [XmlAttribute(AttributeName = "status")]
+        public string FlowStatus { get; set; }
+
+        [XmlIgnore] private double elapsedTime;
+
+        [XmlAttribute(AttributeName = "time")]
+        public double ElapsedTime
+        {
+            get { return Math.Round(elapsedTime, 2); }
+            set { elapsedTime = value; }
+        }
+
+        [XmlAttribute(AttributeName = "classname")]
+        public string classname { get; set; }
+
+        public Failure failure { get; set; }
+
+    }
+
+    public class Failure
+    {
+        public Failure() { }
+        public Failure(string stacktrace)
+        {
+            Message = stacktrace;
+            Type = "dummy";
+        }
+
+        [XmlAttribute(AttributeName = "message")]
+        public string Message { get; set; }
+
+        [XmlAttribute(AttributeName = "type")]
+        public string Type { get; set; }
+    }
+
+    public class InvalidSchedule
+    {
+        public String Name
+        {
+            get; private set;
+        }
+        public String StackTrace
+        {
+            get; private set;
+        }
+
+        public InvalidSchedule(String name, String stackTrace)
+        {
+            this.Name = name;
+            this.StackTrace = stackTrace;
+        }
+    }
+
+    public class Messages
+    {
+
+
+        public static readonly String SCHEDULE_FORMAT = "{0}[{1}]";
+        public static readonly String SCHEDULE_DETECTED = "Schedule {0}[{1}] successfully detected!";
+        public static readonly String SCHEDULE_RUN_SUCCESS = "Schedule {0}[{1}] Launched Successfully!";
```

Comment:
> Schedule {0}[{1}] launched successfully

use normal sentence formatting

Reply:
this is used as String formatting in script, so I haven't worked on it.